core/sys/windows/user32.odin

```diff
@@ -568,3 +580,12 @@ RedrawWindowFlags :: enum UINT {
 	RDW_FRAME           = 0x0400,
 	RDW_NOFRAME         = 0x0800,
 }
+
+WinEventFlags :: bit_set[WinEventFlag; DWORD]
+
+WinEventFlag :: enum DWORD {
+    WINEVENT_OUTOFCONTEXT   = 0,
```

Comment:
I should've been clearer that bit sets use 1 << flag, where `0` becomes `1`, and so forth. That's why I suggested that `WINEVENT_OUTOFCONTEXT` would not be present if used in a `bit_set`. The equivalent to `WINEVENT_OUTOFCONTEXT` as the zero value would be `{}`.

`WINEVENT_OUTOFCONTEXT | WINEVENT_SKIPOWNTHREAD` would become `{.SKIPOWNTHREAD}`, with out of context implied.

As such the enum should be:
```cpp
// OUTOFCONTEXT is the zero value, use {}
WinEventFlag :: enum DWORD {
    SKIPOWNTHREAD  = 0,
    SKIPOWNPROCESS = 1,
    INCONTEXT      = 2,
}
```

The Battery status `bit_set` is correct. I'd already made the adjustment in the example code I suggested.

Reply:
oh that makes sense, I overlooked that